django/db/models/sql/compiler.py

```diff
@@ -355,19 +356,18 @@ def compile(self, node, select_format=False):
             sql, params = vendor_impl(self, self.connection)
         else:
             sql, params = node.as_sql(self, self.connection)
-        if select_format and not self.subquery:
+        if select_format is FORCE or (select_format and not self.query.subquery):
```

Comment:
This was to fix an Oracle GIS test: 194ecc08d133727faf0fa3f46db086934f203b7a. There's probably a cleaner solution, but I couldn't come up with anything quickly (even this fix took awhile). The `select_format` is required in an aggregate query, even if `query.subquery` is `True`.